### PR TITLE
Minor style enhancements

### PR DESCRIPTION
**Pull request by joseangelbujalance**

### CODE REVIEW

File: src/main/java/io/github/mivek/facade/MetarFacade.java [hunk 1]
```diff
@@ -45,11 +45,8 @@ public Metar retrieveFromAirport(final String pIcao) throws IOException, ParseEx
         URL url = new URL(website);
         URLConnection urlCo = url.openConnection();
         try (BufferedReader br = new BufferedReader(new InputStreamReader(urlCo.getInputStream()))) {
-            String line = (String) br.lines().toArray()[1];
-            br.close();
```
Comment (joseangelbujalance) [inline src/main/java/io/github/mivek/facade/MetarFacade.java]: There's no need to manually close an `AutoCloseable` implementation that is defined in a try-with-resources block

File: src/main/java/io/github/mivek/facade/MetarFacade.java [hunk 2]
```diff
@@ -45,11 +45,8 @@ public Metar retrieveFromAirport(final String pIcao) throws IOException, ParseEx
         URL url = new URL(website);
         URLConnection urlCo = url.openConnection();
         try (BufferedReader br = new BufferedReader(new InputStreamReader(urlCo.getInputStream()))) {
-            String line = (String) br.lines().toArray()[1];
-            br.close();
+            String line = br.lines().toArray(String[]::new)[1];
             return getParser().parse(line);
-        } catch (Exception e) {
```
Comment (joseangelbujalance) [inline src/main/java/io/github/mivek/facade/MetarFacade.java]: Haven't understood this

File: src/main/java/io/github/mivek/facade/TAFFacade.java [hunk 3]
```diff
@@ -46,13 +46,9 @@ public TAF retrieveFromAirport(final String pIcao) throws IOException, URISyntax
...
         try (BufferedReader br = new BufferedReader(new InputStreamReader(url.openStream()))) {
             StringBuilder sb = new StringBuilder();
-            String inputLine;
             // Throw the first line since it is not part of the TAF event.
             br.readLine();
-            while ((inputLine = br.readLine()) != null) {
-                // Remove tabs and add line returns.
-                sb.append(inputLine.replaceAll("\\s{2,}", "")).append("\n");
-            }
+            br.lines().forEach(currentLine -> sb.append(currentLine.replaceAll("\\s{2,}", "")).append("\n"));
```
Comment (joseangelbujalance) [inline src/main/java/io/github/mivek/facade/TAFFacade.java]: Should have kept the comment
```suggestion
            // Remove tabs and add line returns.
            br.lines().forEach(currentLine -> sb.append(currentLine.replaceAll("\\s{2,}", "")).append("\n"));
```

File: src/main/java/io/github/mivek/facade/MetarFacade.java [hunk 4]
```diff
@@ -45,11 +45,8 @@ public Metar retrieveFromAirport(final String pIcao) throws IOException, ParseEx
         URL url = new URL(website);
         URLConnection urlCo = url.openConnection();
         try (BufferedReader br = new BufferedReader(new InputStreamReader(urlCo.getInputStream()))) {
-            String line = (String) br.lines().toArray()[1];
```
Comment (joseangelbujalance) [inline src/main/java/io/github/mivek/facade/MetarFacade.java]: You can get rid of the cast providing a generator function

Comment (mivek) [reply src/main/java/io/github/mivek/facade/MetarFacade.java]: 👍

**Pull request merged**